### ISSUE TITLE
DNSCrypt won't open in new windows update, please fix it.

### ISSUE THREAD

**Issue by jiemmy4free** [bitbeans/SimpleDnsCrypt#433]
DNSCrypt won't open in new windows update, please fix it.

**Comment by Kaan88** [comment]
I uninstalled the latest cumulative update, kb4515384. SimpleDnsCrypt wasn't the only app that stopped working for me. (Potplayer, NoteBook FanControl are the ones that stopped working.)

**Comment by kempol181** [comment]
> 
> 
> I uninstalled the latest cumulative update, kb4515384. SimpleDnsCrypt wasn't the only app that stopped working for me. (Potplayer, NoteBook FanControl are the ones that stopped working.)

thanks you for this, mine doesnt work all of a sudden. uninstalling the update works.

**Comment by rockyprabowo** [comment]
I can confirm that I have this issue.
The crash was caused by d3d9.dll which is a part of DirectX library. Some application are affected with the same crash signature too, but it isn't as crucial as SimpleDNSCrypt for me.
Uninstalling cumulative updates or doing a in-place upgrade with ISO from August fixed the issue.
```
<?xml version="1.0" encoding="UTF-16"?>
<WERReportMetadata>
	<OSVersionInformation>
		<WindowsNTVersion>10.0</WindowsNTVersion>
		<Build>18362</Build>
		<Product>(0x30): Windows 10 Pro</Product>
		<Edition>Professional</Edition>
		<BuildString>18362.329.amd64fre.19h1_release.190318-1202</BuildString>
		<Revision>329</Revision>
		<Flavor>Multiprocessor Free</Flavor>
		<Architecture>X64</Architecture>
		<LCID>1033</LCID>
	</OSVersionInformation>
	<ProblemSignatures>
		<EventType>APPCRASH</EventType>
		<Parameter0>SimpleDnsCrypt.exe</Parameter0>
		<Parameter1>0.6.6.0</Parameter1>
		<Parameter2>5cf5ec69</Parameter2>
		<Parameter3>d3d9.dll</Parameter3>
		<Parameter4>10.0.18362.329</Parameter4>
		<Parameter5>1ac9a115</Parameter5>
		<Parameter6>c0000005</Parameter6>
		<Parameter7>00000000000ac44e</Parameter7>
	</ProblemSignatures>
	
	<SystemInformation>
		<SystemManufacturer>ASUSTeK COMPUTER INC.</SystemManufacturer>
		<SystemProductName>VivoBook_ASUSLaptop X570ZD_F570ZD</SystemProductName>
		<BIOSVersion>X570ZD.310</BIOSVersion>
		<OSInstallDate>1569377396</OSInstallDate>
		<OSInstallTime>2019-09-25T02:09:54Z</OSInstallTime>
		<TimeZoneBias>-07:00</TimeZoneBias>
	</SystemInformation>
	<SecureBootState>
		<UEFISecureBootEnabled>0</UEFISecureBootEnabled>
	</SecureBootState>
</WERReportMetadata>
```

**Comment by SanderBouwhuis** [comment]
I just now installed Simple DnsCrypt for the very first time, but nothing happens. When I start it from a command prompt (even with elevated/admin rights) the process exits immediately without anything happening.

How can I fix this?

**Comment by rockyprabowo** [comment]
Looks like this issue has been fixed at least on Windows 10 build 18362.418.
Now SimpleDNSCrypt and other applications that have this issue before can launch properly.

![image](https://user-images.githubusercontent.com/12021586/69692159-c079c000-1103-11ea-9930-dc98e99bb206.png)
